src/index.ts

```diff
@@ -20,5 +20,7 @@ export {
   SWRHook,
   Fetcher,
   MutatorCallback,
-  Middleware
+  Middleware,
+  ValueKey,
+  Result
```

Comment:
Are those 2 types necessary for the users? Or is it just for `useSWRInfinite`?

Reply:
Currently it is just for ```useSWRInfinite ```. 
But it might be useful for user to build middleware like ```useSWRMutation```

Reply:
Makes sense, thanks!